### ISSUE TITLE
NVDA becomes mute after reading a link

### ISSUE THREAD

**Issue by juliemoynat** [WordPress/gutenberg#11975]
**Describe the bug**
I was navigating through a text in Gutenberg editor with the NVDA screen reader and right arrow key (so that it can read text letter by letter). Then, I was arriving in a link text. And suddenly, NVDA becomes mute, in Gutenberg, in my browser, in all my computer.
I need to restart NVDA to make it speak again.

**To Reproduce**
1. Open NVDA and a browser (Firefox, Chrome, Edge)
2. Open a page with Gutenberg editor with text and links inside the block
3. Go inside this block and navigate with the right arrow key
4. NVDA reads text letter by letter as soon as you get the link: it's now mute!

**Expected behavior**
NVDA doesn't become mute when it reads a link.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browsers: Firefox, Edge, Chrome
 - Version: last versions
 - Screen reader: NVDA 2018.3.2

**Additional context**
- I've found this bug on Gutenberg 4.4.0 (can be reproduced here: https://frontenberg.tomjn.com/ ) but it was not present in Gutenberg 4.0.0 (you can test here https://testgutenberg.com/ ). I don't know if the bug is new in 4.4.0.
- I tried to reproduce this bug with another computer with the same configuration and I didn't reproduce it.
- I've uninstalled and then reinstalled NVDA and the bug is still there.
- I've no idea from where is the problem but I prefer to make a ticket for nothing than missing out on this bug (that's really strange that it works in 4.0.0 and not in 4.4.0).

**Comment by tofumatt** [comment]
That's very strange indeed. I haven't seen this in master when testing locally or on Frontenberg. You mentioned you couldn't reproduce it on any other machine—I'm wondering if there's something up with your local setup, because I couldn't get this to happen for me 😢

If you could provide steps to reproduce we should re-open this bug, but because I can't reproduce it I'm not sure it's an issue with Gutenberg itself. Thanks for reporting it though! 😊

**Comment by juliemoynat** [comment]
For information, here is my NVDA log when the problem occured:

```
ERROR - unhandled exception (13:44:13.332):
Traceback (most recent call last):
  File "_ctypes/callbacks.c", line 315, in 'calling callback function'
  File "synthDrivers\oneCore.pyo", line 231, in _callback
ValueError: invalid literal for int() with base 10: ''
```

**Comment by tofumatt** [comment]
Thanks for the log @juliemoynat!

I hate to shift the responsibility but that reads like an NVDA bug. There was some discussion in Slack that this is likely an NVDA issue: https://wordpress.slack.com/archives/C02RP4X03/p1542383096718600

😢

**Comment by juliemoynat** [comment]
Yes, I'm in the discussion in Slack ;)
But the issue linked in Slack is not the same.

That's really strange that I haven't the problem in Gutenberg 4.0.0 but in 4.4.0. So I'm not really sure it's a NVDA problem. An other French user can reproduce the problem too.

I'm waiting for other tests from other people. Maybe the log can help to know if it's a NVDA problem or not.
